crates/pop-cli/src/common/contracts.rs

```diff
@@ -89,6 +89,33 @@ pub fn has_contract_been_built(path: Option<&Path>) -> bool {
 		.unwrap_or_default()
 }
 
+/// Requests and collects function arguments from the user via CLI interaction.
+///
+/// # Arguments
+/// * `function` - The contract function containing argument definitions.
+/// * `cli` - Command line interface implementation for user interaction.
+///
+/// # Returns
+/// A vector of strings containing the user-provided argument values.
+pub fn request_contract_function_args(
```

Comment:
This function can have its own unit test that tests the CLI output. 
As an example the one below:
https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-cli/src/common/contracts.rs#L146